src/helm/benchmark/static/schema.yaml

```diff
@@ -2769,6 +2769,40 @@ run_groups:
       when: n/a
       language: synthetic
 
+  - name: med_qa
+    display_name: MedQA
+    description: MedQA is an open domain question answering dataset composed of questions from professional medical board exams ([Jin et al. 2020](https://arxiv.org/pdf/2009.13081.pdf)).
+    metric_groups:
+      - accuracy
+      - efficiency
+      - general_information
+    environment:
+      main_name: quasi_exact_match
+      main_split: test
```

Comment:
FYI: These is pre-existing issue that the instances used in `helm-run` will be both from `valid` and `test` splits, but the `helm-summarize` will only use the instances from the `test`  split.